_episodes/07-find.md

```diff
@@ -662,11 +662,11 @@ survived so long because it is one of the most productive programming
 environments ever created --- maybe even *the* most productive. Its syntax
 may be cryptic, but people who have mastered it can experiment with
 different commands interactively, then use what they have learned to
-automate their work. Graphical user interfaces may be better at the
-first, but the shell is still unbeaten at the second. And as Alfred
-North Whitehead wrote in 1911, 'Civilization advances by extending the
-number of important operations which we can perform without thinking
-about them.'
+automate their work. Graphical user interfaces may be easier to use at
+first, but the once learned the productivity in the shell is unbeatable. 
```

Comment:
I think after "once learned" we need a comma.

Reply:
transform to "but once learned,"

Reply:
Thanks